youtube_dl/extractor/mixcloud.py

```diff
@@ -122,3 +120,31 @@ def _real_extract(self, url):
             'view_count': view_count,
             'like_count': like_count,
         }
+
+class ServerAttemptSequence:
```

Comment:
@jaimeMF Thanks for your suggestions. I prefer your second solution and implemented it here: https://github.com/Roman2K/youtube-dl/commit/f4fd09ffd5d1e9f218267b2206a90aa998cbcd9d#diff-a845099a3da4a695771782a299402291R125 -- what do you think?